powerscale/provider/nfs_export_settings_resource_test.go

```diff
@@ -222,6 +238,12 @@ resource "powerscale_nfs_export_settings" "test" {
 }
 `
 
+var nfsExportSettingsSnapshotResourceConfig = `
+resource "powerscale_nfs_export_settings" "test" {
+	snapshot = "-"
```

Comment:
Should we mention `-` meaning `default` in doc?

Reply:
Good point! 
Modified.